test/integration/depots.js

```diff
@@ -53,93 +53,83 @@ describe('(/depots) The depots API ', () => {
 
   it('POST /depots create a new depot in the database', () => {
     return agent.post('/depots')
-    .send(newDepot)
-    .then((res) => {
-      helpers.api.created(res);
-      expect(res.body.uuid).to.equal(newDepot.uuid);
-    })
-    .catch(helpers.handler);
+      .send(newDepot)
+      .then((res) => {
+        helpers.api.created(res);
+        expect(res.body.uuid).to.equal(newDepot.uuid);
+      })
+      .catch(helpers.handler);
   });
 
   it('POST /depots create a second depot (without sending uuid)', () => {
     return agent.post('/depots')
-    .send(secondDepot)
-    .then((res) => {
-      helpers.api.created(res);
-    })
-    .catch(helpers.handler);
+      .send(secondDepot)
+      .then((res) => {
+        helpers.api.created(res);
+      })
+      .catch(helpers.handler);
   });
 
   it('POST /depots should not create when missing data', () => {
     delete badDepot.text;
     return agent.post('/depots')
-    .send(badDepot)
-    .then((res) => {
-      helpers.api.errored(res, 400);
-      expect(res.body.code).to.be.equal('ERRORS.ER_NO_DEFAULT_FOR_FIELD');
-    })
-    .catch(helpers.handler);
+      .send(badDepot)
+      .then((res) => {
+        helpers.api.errored(res, 400);
+        expect(res.body.code).to.be.equal('ERRORS.ER_NO_DEFAULT_FOR_FIELD');
+      })
+      .catch(helpers.handler);
   });
 
   it('PUT /depots update an existing depot', () => {
     return agent.put(`/depots/${newDepot.uuid}`)
-    .send(editDepot)
-    .then((res) => {
-      expect(res).to.have.status(200);
-      expect(res.body[0].text).to.exist;
-      expect(res.body[0].text).to.be.equal(editDepot.text);
-      expect(res.body[0].is_warehouse).to.exist;
-      expect(res.body[0].is_warehouse).to.be.equal(editDepot.is_warehouse);
+      .send(editDepot)
+      .then((res) => {
+        expect(res).to.have.status(200);
+        expect(res.body[0].text).to.be.equal(editDepot.text);
```

Comment:
One way to improve this code (not saying you need to do it in the PR), is to do the following:
```js
const [updated] = res.body;

expect(updated.text).to.equal(editDpot.text); 
// .... etc ....
```

It just makes it easier to read.